docs/codeql/codeql-cli/creating-codeql-databases.rst

```diff
@@ -165,13 +165,14 @@ build steps, you may need to explicitly define each step in the command line.
 
 .. pull-quote:: Creating databases for Go
   
-   For Go, you should always use the CodeQL autobuilder. Install the Go
-   toolchain (version 1.11 or later) and, if there are dependencies, the
-   appropriate dependency manager (such as `dep
+   For Go, install the Go toolchain (version 1.11 or later) and, if there
+   are dependencies, the appropriate dependency manager (such as `dep
    <https://golang.github.io/dep/>`__).
    
-   Do not specify any build commands, as you will override the autobuilder
-   invocation, which will create an empty database.  
+   The Go autobuilder attempts to automatically detect Go code in a repository,
+   and only runs build scripts in an attempt to fetch dependencies. To force
+   CodeQL to use your build script, set the environment variable
+   `CODEQL_EXTRACTOR_GO_BUILD_TRACING=on` or pass a command.
```

Comment:
Maybe it's just me, but I was a tiny bit confused by "or pass a command" when first reading this.
Which "command" line argument are you referring to? Why is the command I should pass not named here ("a command"?
But then I realized that you meant `--command`.

Feel free to ignore all of my three suggestions, if it's just me who is confused ^^

```suggestion
   `CODEQL_EXTRACTOR_GO_BUILD_TRACING=on` or use the ``--command`` option.
```

```suggestion
   `CODEQL_EXTRACTOR_GO_BUILD_TRACING=on` or pass a command using the ``--command`` option.
```

```suggestion
   `CODEQL_EXTRACTOR_GO_BUILD_TRACING=on`. Build tracing is also enabled when using ``codeql`` with the ``--command`` option.
```

Reply:
I think it's always good to be more clear and your first suggestion looks great to me. ✨  Thanks for catching that confusing language.

Reply:
- [ ]